cranelift-wasm/src/state.rs

```diff
@@ -130,8 +130,13 @@ impl ControlStackFrame {
 /// - The depth of the two unreachable control blocks stacks, that are manipulated when translating
 ///   unreachable code;
 pub struct TranslationState {
+    /// A stack of values corresponding to the active values in the input wasm function at this
```

Comment:
I was initially thinking we'd want to expose the function's corresponding `TranslationState` (in case a user of cranelift-wasm wanted to add/modify values or the control stack), but that ended up not being necessary in how I'm using this right now. `#[deny(missing_docs)]` got me to fill in docs here and I figure they don't hurt to keep around :)

It seems like tweaking `TranslationState` would be a very typical `{before,after}_translate_operator` use case, so passing along `TranslationState` might make sense even if not strictly necessary in my use right now